Makefile

```diff
@@ -39,6 +39,16 @@ compile_go_mock_yield_token:
 
 compile_mock_yield_token: compile_solidity_mock_yield_token compile_go_mock_yield_token
 
+compile_solidity_mock_swivel_token:
```

Comment:
swivel token?

swivel is a contract / protocol not a token.  yes there should be `compile...swivel` just not with `_token`.